coriolis/tests/api/middleware/test_auth.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2022 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from unittest import mock
+
+from oslo_middleware import request_id
+import webob
+
+from coriolis.api.middleware import auth
+from coriolis.api.middleware.auth import CoriolisKeystoneContext
+from coriolis.api import wsgi
+from coriolis import context
+from coriolis.tests import test_base
+
+
+class CoriolisTestException(Exception):
+    pass
+
+
+class CoriolisKeystoneContextTestCase(test_base.CoriolisBaseTestCase):
+    """Test suite for the Coriolis api middleware auth."""
+
+    def setUp(self):
+        super(CoriolisKeystoneContextTestCase, self).setUp()
+        self.auth = auth.CoriolisKeystoneContext(wsgi.Middleware)
+
+    @mock.patch.object(context, "RequestContext")
+    @mock.patch.object(CoriolisKeystoneContext, "_get_project_id")
+    @mock.patch.object(CoriolisKeystoneContext, "_get_user")
+    def test__call__(
+        self,
+        mock_get_user,
+        mock__get_project_id,
+        mock_request_context
+    ):
+        req_mock = mock.Mock()
+        mock__get_project_id.return_value = 'mock_tenant'
```

Comment:
You don't need to assign a value to these mocks. Instead, please pass `mock__get_project_id.return_value` and `mock_get_user.return_value` to `mock_request_context.assert_called_once_with()` call.